<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WPresBalance</class>
 <widget class="QWidget" name="WPresBalance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>175</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>175</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WPresBalance</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(128, 128, 128);</string>
  </property>
  <widget class="QLabel" name="lbBALIT_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>32</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>PT6861A</string>
   </property>
  </widget>
  <widget class="WLabel" name="lbBALI_1">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>32</y>
     <width>38</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>11.30</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbBALIU_1">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>32</y>
     <width>38</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>PSIA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBALIU_2">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>50</y>
     <width>38</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>PSIA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbBALIT_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>PT6861B</string>
   </property>
  </widget>
  <widget class="WLabel" name="lbBALI_2">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>50</y>
     <width>38</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="text">
    <string>11.60</string>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="Alineacion" stdset="0">
    <enum>WLabel::Centro</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbFondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btCerrar">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>173</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192, 192, 192);</string>
   </property>
   <property name="text">
    <string>PRES BALANCE IMPULSO</string>
   </property>
  </widget>
  <zorder>lbFondo</zorder>
  <zorder>lbBALIT_1</zorder>
  <zorder>lbBALI_1</zorder>
  <zorder>lbBALIU_1</zorder>
  <zorder>lbBALIU_2</zorder>
  <zorder>lbBALIT_2</zorder>
  <zorder>lbBALI_2</zorder>
  <zorder>btCerrar</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WLabel</class>
   <extends>QWidget</extends>
   <header>wlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
